internal/runbits/requirements/requirements.go

```diff
@@ -79,24 +90,32 @@ func executePackageOperation(prime primeable, packageName, packageVersion string
 		}()
 	} else {
 		language, err := model.LanguageByCommit(pj.CommitUUID())
-		if err == nil {
-			langName = language.Name
+		if err != nil {
+			return locale.WrapError(err, "err_package_get_language", "Could not get language from project")
+		}
```

Comment:
You're introducing a new error here. This might be desirable for the language operation, but not so for regular packages. I think this will break the behavior on line 109 as it can never get there through this path.

Reply:
This was just a reaction to an unhandled error. Updated now to at least log the error.